src/MassiveAction.php

```diff
@@ -759,7 +759,7 @@ public static function getAllMassiveActions($item, $is_deleted = false, ?CommonD
             }
 
             // Specific actions
-            $actions += $item->getSpecificMassiveActions($checkitem);
+            $actions += $item->getSpecificMassiveActions($checkitem, $items_id);
```

Comment:
Is it a good idea to call method with 2 arguments where only 1 is required in almost every case?